torch/jit/_script.py

```diff
@@ -0,0 +1,169 @@
+"""TorchScript
+
+This module contains functionality to support the JIT's scripting frontend, notably:
+    - torch.jit.script
+
+This is not intended to be imported directly; please use the exposed
+functionalities in `torch.jit`.
+"""
+import torch
+
+import functools
+import collections
+
+from torch.jit._recursive import ScriptMethodStub
+from torch.nn import Module
+from torch.jit._state import _enabled
+from torch._six import with_metaclass
+
+if _enabled:
+    Attribute = collections.namedtuple("Attribute", ["value", "type"])
+else:
+
+    def Attribute(value, type):
+        return value
+
+
```

Comment:
reason why `script` is not defined in this file ?

Reply:
I was limiting this first PR to the things necessary to move tracing out, just to keep things manageable. I can either roll the scripting-related changes into this one or do it in a followup.